### ISSUE TITLE
@tag for accessibility settings

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#162459]
It could be helpful to have a single place to see all accessibility-related settings. We could add a `@tag:accessibility` to the settings editor which could show audio cues, font sizes, color customizations, zoom level, contrast ratio, editor accessibility, anything else mentioned in https://code.visualstudio.com/docs/editor/accessibility

cc @isidorn 

This should maybe also be discoverable in another way, like a menu entry under the gear icon, or the user could be prompted to see this page after enabling accessibility mode in the editor

**Comment by meganrogge** [comment]
I don't see any downsides of doing this (only upsides).

**Comment by meganrogge** [comment]
Verify that `workbench.action.openAccessibilitySettings` works well for you and that the accessibility tag is useful/contains the settings you'd expect

**Comment by joyceerhl** [comment]
@meganrogge Can extensions participate in this? Live Share has an entire setting `liveshare.accessiblity.accessibilityFeaturesEnabled`

**Comment by meganrogge** [comment]
@rzhao271 you're more familiar w the settings editor - do you know if extensions can add our tags?

**Comment by rzhao271** [comment]
Extensions should be able to participate.
For example, `typescript.disableAutomaticTypeAcquisition` has the `usesOnlineServices` tag.

**Comment by meganrogge** [comment]
@joyceerhl that's a good point and a reason this will be in the release notes
